app/Models/Loggable.php

```diff
@@ -100,27 +133,65 @@ private function determineLogItemType($log)
      */
     public function logCheckin($target, $note)
     {
+        $settings = Setting::getSettings();
         $log = new Actionlog;
         $log->target_type = get_class($target);
-        $log->target_id = $target->id;
+
+        if ($target) {
+            $log->target_id = $target->id;
+        } else {
+            $target = null;
```

Comment:
Should this be log->target_id = null? THe only way we hit the else branch is if the target is null to begin with, right?